modMcf/test/org/aion/mcf/account/KeystoreTest.java

```diff
@@ -195,4 +193,43 @@ public void testList() {
 
         assertTrue(hasAddr);
     }
+
+    @Test (expected = NullPointerException.class)
+    public void testBackupAccountWithNullInput(){
+        Keystore.backupAccount(null);
+    }
+
```

Comment:
The `list` functionality could be tested better. I know you didn't write that test but its for-loop doesn't make much sense (testing if the list is size 1 and contains an address). But it would be good to also test this when you expect no entries and when you expect multiple entries.